sky/backends/cloud_vm_ray_backend.py

```diff
@@ -72,10 +72,10 @@
 _WORKER_IP_MAX_ATTEMPTS = 5
 
 _TEARDOWN_FAILURE_MESSAGE = (
-    f'{colorama.Fore.RED}Failed to terminate '
+    f'\n{colorama.Fore.RED}Failed to terminate '
     '{cluster_name}. {extra_reason}'
     'If you want to ignore this error and remove the cluster '
-    'from from Sky\'s status table, use `sky down --purge`.'
+    'from the status table, use `sky down --purge`.'
```

Comment:
Run `sky down --purge` to bypass/ignore errors when removing the cluster.

Reply:
The current wording puts the purpose first and action next, which reads clearer to me.